src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -28,7 +28,8 @@
 SSLERROR_TEMPLATE = ('Certificate verification failed. This typically happens when using Azure CLI behind a proxy '
```

Comment:
`SSLERROR_TEMPLATE` contains both error itself and recommendation, which makes this line unreasonable:

https://github.com/Azure/azure-cli/blob/b6aa49936fd538ddd120438b396b570878e078e1/src/azure-cli-core/azure/cli/core/util.py#L85-L87

@houk-ms, should we split them?

Reply:
yeah, I think it's better if we can